src/clients/ClientSettings.tsx

```diff
@@ -130,40 +120,121 @@ export const ClientSettings = ({
               />
             </FormGroup>
             <FormGroup
-              label={t("webOrigins")}
-              fieldId="kc-web-origins"
+              label={t("validRedirectUri")}
+              fieldId="kc-redirect"
               labelIcon={
                 <HelpItem
-                  helpText="clients-help:webOrigins"
-                  fieldLabelId="clients:webOrigins"
+                  helpText="clients-help:validRedirectURIs"
```

Comment:
Small peeve, this is not capitalized elsewhere.
```suggestion
                  helpText="clients-help:validRedirectUris"
```